__tests__/app.test.js

```diff
@@ -18,3 +24,21 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe('/api/topics', () => {
+  test.only('GET:200 sends an array of topics to the client', () => {
```

Comment:
you need to remove any .only's before pushing because we don't know if your other tests pass. i will still review this one but next time it will get rejected sorry!